commands/curation/curationaudit.go

```diff
@@ -298,7 +310,13 @@ func (ca *CurationAuditCommand) auditTree(tech coreutils.Technology, results map
 		return err
 	}
 	rootNode := fullDependenciesTrees[0]
-	_, projectName, projectScope, projectVersion := getUrlNameAndVersionByTech(tech, rootNode, "", "")
+	_, projectName, projectScope, projectVersion := getUrlNameAndVersionByTech(tech, rootNode, nil, "", "")
+	if projectName == "" {
+		workPath, err := os.Getwd()
+		if err == nil {
+			projectName = filepath.Base(workPath)
+		}
+	}
```

Comment:
```suggestion
	if projectName == "" {
		if workPath, err := os.Getwd(); err == nil {
			projectName = filepath.Base(workPath)
		}
	}
```
what if `err != nil` ?

Reply:
Fixed